.cirrus/tasks.yml

```diff
@@ -140,25 +140,16 @@ chart_testing_task:
     <<: *CONTAINER_TEMPLATE
     cpu: 2
     memory: 4Gb
-    additional_containers:
-      - name: dockerdaemon
-        privileged: true
-        cpu: 4
-        memory: 16Gb
-        image: public.ecr.aws/docker/library/docker:dind
-        port: 2375
-        env:
-          DOCKER_DRIVER: overlay2
-          DOCKER_TLS_CERTDIR: ""
   <<: *CLONE_SCRIPT_TEMPLATE
   start_kind_script:
```

Comment:
We are now connecting to a real test cluster instead of an emulated one :) Shall we rephrase the task's name with something like "authenticate_to_test_cluster"? :)

Reply:
We can let the name of the task as it is because we didn't change what the task is doing.

Reply:
sorry, not the overall task's name, just the step's name to something like "authenticate_to_test_cluster_script". WDYT?

Reply:
also, we should delete the `wait_forkind.sh`... this script won't be used anymore

Reply:
oh yes totally, I will make those changes. Thank you.